addons/website/views/snippets/snippets.xml

```diff
@@ -1230,7 +1233,7 @@
     </div>
 
     <!-- Scroll to next section button (only for full height) -->
-    <div data-js="ScrollButton" data-selector="section" data-exclude="[data-snippet] :not(.oe_structure) > [data-snippet]">
+    <div data-js="ScrollButton" data-selector="section" data-exclude="[data-snippet] :not(.oe_structure) > [data-snippet], .s_instagram_page">
```

Comment:
Why? :open_mouth:

Reply:
![image](https://github.com/odoo/odoo/assets/78849981/a64bd945-a804-4245-86d5-09ca17cabeba)

Reply:
Oh, yes. We still have that bad UX where you can only have a scroll button if you set up a full height... sad.